app/assets/javascripts/school_administrator_dashboard/discipline_dashboard/SchoolDisciplineDashboard.js

```diff
@@ -216,17 +253,6 @@ SchoolDisciplineDashboard.contextTypes = {
 };
 SchoolDisciplineDashboard.propTypes = {
   dashboardStudents: PropTypes.array.isRequired,
-  schoolDisciplineEvents: PropTypes.arrayOf(PropTypes.shape({
-    student_id: PropTypes.number.isRequired, //ID of student involved in incident
-    location: PropTypes.string, //Place where incident occurred
-    time: PropTypes.string, //Time of day for incident - NULL if no specific time recorded
-    classroom: PropTypes.string, //Name of student's homeroom teacher
-    student_grade: PropTypes.string, //Grade of student
-    day: PropTypes.string, //Day of week on which incident occurred
-    offense: PropTypes.string, //Specific type of incident
-    student_race: PropTypes.string, //Race of student
-    occurred_at: PropTypes.string, //Date for incident, used in filtering specific date ranges
-  })).isRequired,
```

Comment:
This seems like good checking for dev/test and documentation of intent.  Is there a way to keep it and say `dashboardStudents` has an array of objects that have the `discipline_events` field in this shape?

Reply:
I think we can do that. Not all students will have this information, but there's no reason we couldn't validate the structure for those that do.